WooCommerce/Classes/System/WooNavigationController.swift

```diff
@@ -30,6 +30,19 @@ class WooNavigationController: UINavigationController {
     override var preferredStatusBarStyle: UIStatusBarStyle {
         return StyleManager.statusBarLight
     }
+
+    override func shouldPopOnBackButton() -> Bool {
+        if let vc = topViewController {
+            return vc.shouldPopOnBackButton()
+        }
+        return shouldPopOnBackButton()
```

Comment:
Should this include `super`?

```suggestion
        return super.shouldPopOnBackButton()
```